pkg/storage/test/tuples.go

```diff
@@ -474,6 +476,98 @@ func TupleWritingAndReadingTest(t *testing.T, datastore storage.OpenFGADatastore
 		_, err = iter.Next()
 		require.ErrorIs(t, err, storage.ErrIteratorDone)
 	})
+
+}
+
+// TupleReadOrder tests the ordering of read and read page.
+// They are for specific datastore implementation and may not be appropriate
+// for all storage implementation (i.e,. memory)
+
+func TupleReadOrder(t *testing.T, datastore storage.OpenFGADatastore) {
```

Comment:
```suggestion
func TupleReadOrderTest(t *testing.T, datastore storage.OpenFGADatastore) {
```

Reply:
Obsolete as we don't have the time provider anymore